metricbeat/docs/modules/sql.asciidoc

```diff
@@ -227,6 +227,51 @@ for each row. For example, this event is created for the first row:
 }
 ----
 
+*Oracle Database Connection Pre-requisites*
```

Comment:
Fix format for title, something like this (depending on the level this title should be):
```suggestion
=== Oracle Database Connection Pre-requisites
```